crates/bevy_ui/src/entity.rs

```diff
@@ -132,6 +156,48 @@ impl Default for ButtonBundle {
     }
 }
 
+/// A UI node that is a button with a texture sheet
+#[derive(Bundle, Clone, Debug)]
+pub struct ButtonSheetBundle {
```

Comment:
I don't like the creation of a dedicated bundle here: this should be done compositionally.